tests-stress/locust/locust.conf

```diff
@@ -0,0 +1,10 @@
+# master.conf in current directory
+locustfile = locust-notifications.py
+host = https://api.staging.notification.cdssandbox.xyz
+users = 6000
+spawn-rate = 200
+
+# headless = true
+# master = true
+# expect-workers = 5
+# run-time = 5m
```

Comment:
The commented parameters will be useful for executing locust in a build pipeline.